platform-server-kernel/src/main/java/uno/cod/platform/server/core/domain/Template.java

```diff
@@ -40,6 +40,6 @@ public void setFileName(String fileName) {
 
     @Override
     public String filePath() {
-        return String.format("/%s/%s", language, fileName);
+        return String.format("%s", fileName);
```

Comment:
`String.format("%s", s)` is `s`, therefore `filePath()` will always return `fileName`, so it is obsolete, and we should consider renaming `fileName` to `filePath`.